### ISSUE TITLE
timeline option to only view media tweets

### ISSUE THREAD

**Issue by Shogunbenny** [dimdenGD/OldTwitter#340]
I know this is maybe not in the focus of OldTwitter, but it's been such a useful plugin returning a lot of functionality to the website for me, figured I'd ask.

**Comment by swoonet** [comment]
Yes! This function as well as not always seeing a retweet you saw already within a certain timeframe was so nice in old tweetdeck.
